### PR TITLE
add openvpn-client readiness and liveness checks

### LINKED ISSUE #3025: Add a liveness probe to the openvpn-client (Addon)
**User Story**
The openvpn-client pod must have a liveness check. This ensures the openvpn-client gets restarted when its not healthy.
We've seem in a recent outage that the pod might reach the "limbo" which required manual intervention.

This only focuses on the openvpn-client which gets deployed into user clusters.

**Acceptance criteria**
- [ ] Add a liveness probe to the OpenVPN client pod

### PR DESCRIPTION
**Pull request by mrIncompetent**
**What this PR does / why we need it**:
Add openvpn-client readiness and liveness checks

Fixes #3025

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by mrIncompetent** [comment]
/hold
I need to update the image tags

File: addons/openvpn/openvpn-client-dep.yaml [hunk 1]
```diff
@@ -22,14 +22,25 @@ spec:
...
+          exec:
+            command:
+              - "/bin/ping"
+              - "-c"
+              - "1"
+              - "-w"
+              - "5"
+              - "10.20.0.1"
+          initialDelaySeconds: 60
         readinessProbe:
```
Comment (alvaroaleman) [inline addons/openvpn/openvpn-client-dep.yaml:35]: Why the readyness probe, what does us help that with?

Comment (thz) [reply addons/openvpn/openvpn-client-dep.yaml:35]: Visibility.

**Comment by mrIncompetent** [comment]
/hold cancel

@thz Image tags have been updated PTAL

**Pull request merged**